RecoHI/HiJetAlgos/plugins/HiFJRhoFlowModulationProducer.cc

```diff
@@ -7,6 +7,7 @@
 #include "DataFormats/Common/interface/View.h"
 #include "DataFormats/JetReco/interface/Jet.h"
 #include "DataFormats/Math/interface/deltaR.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
 #include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
```

Comment:
Is this code only meant to run on miniAOD, meaning on packedCandidates?
In that case, is the include of PFCandidate.h still needed?

Reply:
The include is still needed. I use Reco::PFCandidate as a converter to find the particle type from PDG ID in line 223.